lib/tasks/add_permission_to_access_level.rake

```diff
@@ -18,5 +18,9 @@ task :add_permission_to_access_level,
           ' this task'
   end
 
-  new_permissions.create
+  created_permissions = new_permissions.create
+  users_with_new_permissions = created_permissions.flatten.map(&:user).uniq.sort_by(&:full_name)
+
+  puts "Created #{created_permissions.count} permission records for the following #{users_with_new_permissions.count} "\
+       "users: #{users_with_new_permissions.map(&:full_name)}"
```

Comment:
Handling these permissions and flattening things and stuff seems awkward. Can we just print the log inside the permissioning script directly? It seems simpler. Additionally, should we use `Rails.logger` instead of `puts`?

Reply:
We're already printing these using Rails.logger in the script, the point of doing this via a `puts` here would be to print this message on a remote console where the rake task was invoked. I'll drop it in favour of the inlined Rails log messages.

Reply:
Ahhh I missed that. Neat!